templates/src/objective-c/BoringSSL-GRPC.podspec.template

```diff
@@ -152,6 +152,11 @@
     end
     s.subspec 'Implementation' do |ss|
       ss.header_mappings_dir = 'src'
+
+      ss.resource_bundles = {
+        s.module_name => 'src/objective-c/PrivacyInfo.xcprivacy'
```

Comment:
This podspec is already in 'src/objective-c/' directory, so should be
`s.module_name => 'PrivacyInfo.xcprivacy'`
here.

Reply:
Good catch, @HannahShiSFB!